### DISCUSSION TITLE
Missing the test of missing translations

### DISCUSSION THREAD

**Discussion by MaiklT** [dnnsoftware/Dnn.Platform#3775] [Ideas]
## Description of problem
The Resource File Verification is a good tool to find out what work has to be done. But compared to older DNN versions I am missing the test of missing translations, i.e. if an entry is not there in an existing file. The "Files older than system default" is not helpful in this case, as most of them did not change for years (and still are valid).

## Description of solution
There should be a test for this. Language files are vital for the DNN worldwide success, and every tool given to those who create them helps that more language files are available.

**Comment by sleupold** [comment]
confirm, this is really needed for using the translation editor properly

**Comment by stale** [comment]
We have detected this issue has not had any activity during the last 90  days. That could mean this issue is no longer relevant and/or nobody has  found the necessary time to address the issue. We are trying to keep the  list of open issues limited to those issues that are relevant to the  majority and to close the ones that have become 'stale' (inactive). If no  further activity is detected within the next 14 days, the issue will be  closed automatically.
If new comments are are posted and/or a solution (pull request) is  submitted for review that references this issue, the issue will not be  closed. Closed issues can be reopened at any time in the future. Please remember those participating in this open source project are volunteers  trying to help others and creating a better DNN Platform for all. Thank  you for your continued involvement and contributions!

**Comment by stale** [comment]
This issue has been closed automatically due to inactivity (as mentioned  14 days ago). Feel free to re-open the issue if you believe it is still  relevant.

**Comment by sleupold** [comment]
AFAIK still an issue and needed

**Comment by MaiklT** [comment]
It is still an issue, and it is really important for localizers and translators. It should not be triaged, and it shot not be defined as a minor issue imho.

**Comment by MaiklT** [comment]
How can we re-open this?

**Comment by valadas** [comment]
I have reopened but moved this to a discussion since this is not actionable until we have a developer that actually indicates wanting to tackle this.
